### PR TITLE
bump manticore minimum version due to obj leak

### LINKED ISSUE #392: Sniffing in 2.2.2 causes memory leak & OOM
Hi,
   I run few instances of logstash, since upgrade to 2.2.2 i started to notice permanent crashes caused by OOM errors. Bigger heap only increase interval between OOMs. I checked heap dump and found excessive amount of **org.apache.http.config.Registry** objects. 

Out of 3GB I had 887000 Registries taking 1.4GB of memory.
screen shot 2016-03-14 at 8 39 01 am

I started to monitory tenured memory utilization and amount of Registries object instances on the heap for each logstash instance.

**Number of Registry objects 1GB heap (jmap -gchisto)**

screen shot 2016-03-14 at 8 38 16 am

**Tenured memory utilization 1GB heap (jstat -gcutil)**
screen shot 2016-03-14 at 8 41 41 am

Drops from 85% to few percent indicate watchdog killed process as there was no point to waste CPU due to CMS cycles as memory won't be recycled.

You may notice one series of data on those screenshots that looks pretty fine, that's instance with sniffing set to false, all others had sniffing => true

Once disabled sniffing on all instances the problem is gone.

**Disabled Sniffing: Number of Registry objects 1GB heap (jmap -gchisto)**
screen shot 2016-03-14 at 8 54 32 am

### PR DESCRIPTION
**Pull request by jsvd**
see more in https://github.com/cheald/manticore/issues/45

fixes #392

### CODE REVIEW

File: logstash-output-elasticsearch.gemspec [hunk 1]
```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
 
   if RUBY_PLATFORM == 'java'
     s.platform = RUBY_PLATFORM
-    s.add_runtime_dependency "manticore", '>= 0.5.2', '< 1.0.0'
+    s.add_runtime_dependency "manticore", '>= 0.5.4', '< 1.0.0'
```
Comment (suyograo) [inline logstash-output-elasticsearch.gemspec]: why not pin it to a version?

Comment (suyograo) [inline logstash-output-elasticsearch.gemspec]: also this might create a dependency issue with http-input?

Comment (suyograo) [inline logstash-output-elasticsearch.gemspec]: should work: https://github.com/logstash-plugins/logstash-mixin-http_client/blob/master/logstash-mixin-http_client.gemspec#L21

Comment (jsvd) [inline logstash-output-elasticsearch.gemspec]: and that's why we don't pin a version :D

**Comment by suyograo** [comment]
LGTM

**Comment by suyograo** [comment]
@jsvd can you add more info to the changelog that it actually fixes a memory leak and reference the manticore issue?

**Comment by jsvd** [comment]
done

**Comment by suyograo** [comment]
Thanks. Please merge and publish :)

**Pull request closed without merging**